kernel/src/actions/deletion_vector.rs

```diff
@@ -175,6 +182,17 @@ fn read_u32(cursor: &mut Cursor<Bytes>, endian: Endian) -> DeltaResult<u32> {
     }
 }
 
+/// decode a slice into a u32
+fn slice_to_u32(buf: &[u8], endian: Endian) -> DeltaResult<u32> {
+    let arry = buf
```

Comment:
out of curiosity, why `arry` instead of `array`?

Reply:
gotta save those keystrokes :) Probably better to just use `array`, I'll do that